client/components/SongBarEntry.jsx

```diff
@@ -13,21 +13,19 @@ const SongBarEntry = ({
   return (
     <div
       className={barClass}
-      style={{ height: Math.floor(height * bar) }}
+      style={{ height: Math.floor(height * barHeight) }}
       onClick={() => handleClick(barFraction)}
-      onPointerOver={() => handleScan(number)}
+      onPointerOver={() => handleScan(barFraction)}
     />
   );
 };
 
 SongBarEntry.propTypes = {
   position: PropTypes.string.isRequired,
-  bar: PropTypes.number.isRequired,
-  number: PropTypes.number.isRequired,
+  barHeight: PropTypes.number.isRequired,
   playState: PropTypes.object.isRequired,
   barFraction: PropTypes.number.isRequired,
   handleScan: PropTypes.func.isRequired,
-  handleExit: PropTypes.func.isRequired,
   handleClick: PropTypes.func.isRequired,
 };
 
```

Comment:
consider having default prop values - i used them as a way to pass some test if some data is not passed through